Source/ImageGallery/ImageGalleryViewCell.swift

```diff
@@ -31,3 +31,22 @@ class ImageGalleryViewCell: UICollectionViewCell {
     imageView.image = image
   }
 }
+
+extension ImageGalleryViewCell {
+  func changeSelectedStatus(selected: Bool? = false, ani: Bool? = true) {
```

Comment:
Please, expand to `func changeSelectedStatus(selected: Bool? = false, animated: Bool? = true)` to match Apple's API.